src/backend/distributed/planner/multi_physical_planner.c

```diff
@@ -2017,7 +2027,6 @@ SubquerySqlTaskList(Job *job)
 
 	/* get list of all range tables in subquery tree */
 	ExtractRangeTableRelationWalker((Node *) subquery, &rangeTableList);
-	anchorRangeTableId = AnchorRangeTableId(rangeTableList);
```

Comment:
We have one more occurrence of `AnchorRangeTableId()`. Would it lead to such performance problems on other cases?